integrations/common/vars.go

```diff
@@ -28,6 +30,24 @@ type OAuthData struct {
 	TokenType    string `var:"oauth_token_type"`
 }
 
+// CheckOAuthToken returns a warning status if OAuthAccessTokenVar is missing in sdktypes.Vars; otherwise, it returns OK.
+// It depends on sdktypes.Vars being preloaded with OAuthAccessTokenVar, which isn't validated.
+func CheckOAuthToken(vs sdktypes.Vars) (sdktypes.Status, error) {
+	if vs.GetValue(OAuthAccessTokenVar) == "" {
+		return sdktypes.NewStatus(sdktypes.StatusCodeWarning, "Init required"), nil
+	}
+	return sdktypes.NewStatus(sdktypes.StatusCodeOK, "Using OAuth 2.0"), nil
+}
+
+// CheckOAuthToken returns a warning status if LegacyOAuthAccessTokenVar is missing in sdktypes.Vars; otherwise, it returns OK.
+// It depends on sdktypes.Vars being preloaded with LegacyOAuthAccessTokenVar, which isn't validated.
```

Comment:
Incorrect function name + https://go.dev/doc/comment#doclinks
```suggestion
// CheckLegacyOAuthToken returns a warning status if [LegacyOAuthAccessTokenVar] is missing in [sdktypes.Vars]; otherwise,
// it returns OK. It depends on [sdktypes.Vars] being preloaded with [LegacyOAuthAccessTokenVar], which isn't validated.
```